apps/mux/src/components/playercode.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { Textarea, CopyButton, TextLink, Box } from '@contentful/f36-components';
 
 interface GetPlayerCodeProps {
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
```

Comment:
instead of `any` would it be better to cast this to something like

```typescript
interface GetPlayerCodeProps {
  params: { [key: string]: string | boolean }
}
```
understand if this isn't warranted

Reply:
Yeah I agree that any types are not ideal. For the scope of this PR, I didn't want to adjust too many things, hence why I made the call to just ignore this line (the rest of the app is enforced with 'no any types'). I played around with the types of this `params` props itself, and I noticed that it is similar to what you suggest as an array of objects, but am worried about any variation not accounted for - unsure if original developer placed the any type to account for variation or not. 🤷‍♀️